package.json

```diff
@@ -12,6 +12,7 @@
     "dev:preset": "./tools/scripts/dev.preset.sh",
     "dev:preset:sync": "tsc -p ./tools/preset-sync/tsconfig.json && node ./tools/preset-sync/index.js",
     "build": "next build",
+    "build:vercel": "./deploy/scripts/build_assets.sh && next build",
```

Comment:
I would argue against mentioning a particular CI vendor in the command name. The command is performing a generic task, so we should not be that specific when naming it. I suggest changing it to something like: `build:next`, `build:app`, `build:machine`, `build:node`, etc., or modify the build script of `package.json` itself. The last option implies that its usage also should be modified.

Reply:
Yup, I agree. I have done with `build:next` as that seems to be suitably general.

Thank you for the feedback!